### PR TITLE
Adding a section on how to uninstall a custom patch

### PR DESCRIPTION
**Pull request by almarchenko**
## Purpose of this pull request

This pull request (PR) is to add information about how to uninstall a custom patch.

## Affected DevDocs pages

https://devdocs.magento.com/cloud/project/project-patch.html#apply-a-custom-patch

### CODE REVIEW

**Review by billygilbert — APPROVED** [review]

File: src/cloud/project/project-patch.md [hunk 1]
```diff
@@ -205,6 +205,29 @@ To apply and test a custom patch on a Cloud environment:
    {:.bs-callout-info}
    Make sure to test all patches in a pre-production environment. For {{site.data.var.ece}}, you can create new branches with the `magento-cloud environment:branch <branch-name>` CLI command.
 
+## Revert a custom patch
+
+To revert/uninstall a previously applied custom patch:
```
Comment (hguthrie) [inline src/cloud/project/project-patch.md:210]:
```suggestion
To revert or uninstall a previously applied custom patch:
```

**Review by hguthrie — APPROVED** [review]
I just have one suggestion, then I can import changes. Thanks!

**Comment by hguthrie** [comment]
@magento import code to magento-devdocs/devdocs

**Comment by m2-github-services** [comment]
@hguthrie the branch with the code is successfully imported into the `magento-devdocs/devdocs` repository. Branch name: `imported-almarchenko-devdocs-9506`.

**Pull request merged**